demos/coreMQTT_Agent/mqtt_agent_task.c

```diff
@@ -956,44 +963,47 @@ static int prvConnectAndCreateDemoTasks( void * pvParameters )
     /* Set the pParams member of the network context with desired transport. */
     xNetworkContext.pParams = &secureSocketsTransportParams;
 
+    /* Initialize the MQTT context with the buffer and transport interface. */
+    xMQTTStatus = prvMQTTAgentInit();
```

Comment:
Should there be an assert for the return value?

Reply:
I think `if` checks are preferred over asserts here